client/landing/gutenboarding/components/create-site-error/style.scss

```diff
@@ -0,0 +1,58 @@
+@import 'assets/stylesheets/gutenberg-base-styles';
+@import '../../mixins';
+
+.create-site-error__background {
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	z-index: 50;
+	background-color: $white;
+
+	.create-site-error__header {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		height: 56px;
```

Comment:
I think we should be able to get the vertical position of the `W` logo to line up if we set this to `64px` to match the other steps in the flow.